src/many-abci/Cargo.toml

```diff
@@ -44,3 +44,6 @@ tokio = { version = "1.13.0", features = [ "full" ] }
 tracing = "0.1.28"
```

Comment:
According to the `vergen` documentation, you are missing `build = "build.rs"` in the `[package]` section of this file. 

That being said, it seems to work anyway.